### PR TITLE
Suppress `"ALL"` when no suppression config is given

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR extends `SuppressRenderer` to handle the case when configuration is given, but no warnings are specified in its `warnings` property

Before this PR it was assumed that `@SuppressWarnings("ALL")` is used only in the absence of ProtoData configuration. 
But real projects do configure ProtoData in some way, and they add `SuppressRenderer` too. The assumption resulted in generating only `@SuppressWarnings({})` because no specific warnings were configured. Because of that generated Java code which dealt with deprecated proto properties produce many deprecated warnings during the build.

New assumption is this: if the project adds `SuppressRenderer` and does not care about specific warnings, suppress them `"ALL"`.

Other notable changes of this PR:
  * Reusable code was moved to `base` and `core-java` repositories.

### CODE REVIEW

**Review by armiol — CHANGES_REQUESTED** [review]
Please see my comments.

File: compiler/src/main/kotlin/io/spine/server/event/Policy.kt [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.event
```
[the remark this answers is not in the document]
Comment (alexander-yevsyukov) [reply compiler/src/main/kotlin/io/spine/server/event/Policy.kt:27]: This is a preparational refactoring, before I move it to `core-java`.

File: compiler/src/main/kotlin/io/spine/server/query/Querying.kt [hunk 2]
```diff
@@ -24,7 +24,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.protodata
+package io.spine.server.query
```
[the remark this answers is not in the document]
Comment (alexander-yevsyukov) [reply compiler/src/main/kotlin/io/spine/server/query/Querying.kt:27]: I do want to move it to `core-java`, in addition to those things that were already planned to be moved to `base`. 

But I'd like to make it two steps, avoiding manual imports editing later.

File: compiler/src/main/kotlin/io/spine/server/event/Policy.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexander-yevsyukov) [reply compiler/src/main/kotlin/io/spine/server/event/Policy.kt:27]: @armiol,  let's finish with https://github.com/SpineEventEngine/base/pull/753 first, and then I'll update dependency on `base` right in this PR.

**Review by armiol — APPROVED** [review]
Given these things are moved to `core-java` right away, LGTM. 

Also, please do not resolve conversations until the original commenter (in this case it was me) got a chance to read your replies :) it's a bit confusing.

**Comment by armiol** [comment]
I have approved `base`-related changes, BTW.

**Comment by alexander-yevsyukov** [comment]
> Given these things are moved to `core-java` right away, LGTM.

Yeah, I'm going to do it that way. So, tomorrow, there would be a PR in `core-java` and then resolution of this one.

**Review by dmdashenkov — APPROVED** [review]

**Pull request merged**